lib/autocomplete.ts

```diff
@@ -61,9 +53,26 @@ export function autocomplete<T>(
     ...options
   };
 
-  const contentFunc = options.contentFunc!;
+  // Keeps track of the last value as typed by the user.
+  let lastAsTyped = inputElem.value;
+  const lis = dom.onElem(inputElem, 'input', () => lastAsTyped = inputElem.value);
+
+  const contentFunc = () => [
+    dom.autoDispose(lis),
+    dom.forEach(choices, (opt) => (
+      menuItem(
+        () => { inputElem.value = opt; if (options.onClick) { options.onClick(opt); }},
+        opt,
+        dom.data('choice', opt),
```

Comment:
Seems to duplicate `el.textContent` -- is this needed, esp given the simplified string-only interface?